frontend-opal/src/components/PortfolioContainer/PortfolioContainer.js

```diff
@@ -2,41 +2,48 @@ import React, { useEffect } from 'react';
 import { View, SafeAreaView, ScrollView } from 'react-native';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import thunks from 'thunks/portfolio';
+import { Card, CardItem } from 'native-base';
+import thunksUserPortfolioInfo from 'thunks/userPortfolioInfo';
 
 import styles from './styles';
 import ValueInfo from './ValueInfo';
 import AllocationInfo from './AllocationInfo';
 import Instruments from './Instruments';
 
-const PortfolioContainer = ({ onGetPortfolioData }) => {
-  useEffect(() => onGetPortfolioData(), [onGetPortfolioData]);
+const PortfolioContainer = ({ onfetchPortfolioEndpoint }) => {
+  useEffect(() => onfetchPortfolioEndpoint(), [onfetchPortfolioEndpoint]);
 
   return (
-    <>
-      <SafeAreaView>
-        <ScrollView>
-          <View style={styles.portfolioArea}>
-            <ValueInfo />
-            <AllocationInfo />
-            <Instruments />
-          </View>
-        </ScrollView>
-      </SafeAreaView>
-    </>
+    <SafeAreaView>
+      <ScrollView>
+        <View style={styles.portfolioArea}>
+          <Card transparent>
+            <CardItem>
+              <ValueInfo />
+            </CardItem>
```

Comment:
@WenyuD can we move the `CardItem` parts into the components?

Reply:
Sure!

Reply:
> @WenyuD can we move the `CardItem` parts into the components?

Sure!